docs/Parts List.md

```diff
@@ -0,0 +1,78 @@
+# Parts List
+
+## Printed Parts
+To Build Sawppy the Rover, you will need to print the following parts. Quantities and notes are listed.
+
+You will need about 3kg of filament to print all the parts.
+
+| QTY | PART | NOTES |
+|:---:|:----:|:-----:|
+| 88 | Clip2n125 | Print a few extra for spares |
+| 12 | Clip3n20 | Print a few extra for spares |
+| 6 | Wheel Hub | Print a few extra for spares |
+| 10 | LX-16A - Coupler | Print a few extra for spares |
+| 10 | LX-16A - Bracket | |
+| 6 | Wheel | |
+| 4 | Steering Knuckle | |
+| 4 | Body Corner | |
+| 2 | DiffBrace | |
+| 2 | DiffLink | |
+| 1 | DiffLower | |
+| 1 | DiffUpper | |
+| 2 | Rod Support | |
+| 1 | Power Panel | |
```

Comment:
Power panel is specific to voltage meter and switch I used. Other builders won't need to get exactly identical items and will probably need to build their own power control panel to accommodate their meter + switch.

Reply:
I was planning to recreate the power panel in OpenSCAD and make it parametric. I would be happy to upload and include the file once it's finished.